client/src/mutableInput/styles.css

```diff
@@ -0,0 +1,46 @@
+.mi-textarea {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #302C2C;
+  width: 16rem;
+  height: 100%;
+  clear: both;
+  border-radius: 0.5rem;
+  resize: none;
+  overflow: hidden;
+  min-height: 1rem;
+  font-size: 1rem;
+}
+
+.mi-div {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #302C2C;
+  width: 16rem;
+  height: 100%;
+  clear: both;
+  padding: 0 0 0.5rem 1.3rem;
+  margin-bottom: 0;
+  overflow-wrap: break-word;
+}
+
+.boardTitle {
```

Comment:
You use hyphens in styles above, but here camelCase. Choose one style and follow it everywhere